### PR TITLE
Update posts.md

### PR DESCRIPTION
**Pull request by chardaway**
Added content for creating POSTS using UI interface.

I am very sorry that I may have performed an incorrect operation last time, which resulted in incorrect results. So I made the necessary modifications and submitted the application.

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

## Summary by CodeRabbit

- **New Features**
  - Enhanced post creation functionality in DataHub.
  - Users can now create Text and Link posts directly through the UI.
  - Detailed steps provided in the Settings tab for creating posts via UI.
  - Option to create posts using the `createPost` GraphQL mutation remains available.

- **Documentation**
  - Updated documentation to include detailed steps and images for creating posts using the UI.
  - Revised the section headers for clarity.

### CODE REVIEW

File: docs/posts.md [hunk 1]
```diff
@@ -14,8 +14,29 @@ DataHub allows users to make Posts that can be displayed on the app. Currently,
 Anyone can view Posts on the home page. To create Posts, a user must either have the **Create Global Announcements** Privilege, or possess the **Admin** DataHub Role.
 
 ## Using Posts
```
Comment (yoonhyejin) [inline docs/posts.md:16]:
```suggestion
```

File: docs/posts.md [hunk 2]
```diff
@@ -14,8 +14,29 @@ DataHub allows users to make Posts that can be displayed on the app. Currently,
...
+# Create Posts Using the UI
```
Comment (yoonhyejin) [inline docs/posts.md:17]:
```suggestion
### Create Posts Using the UI
```

File: docs/posts.md [hunk 3]
```diff
@@ -14,8 +14,29 @@ DataHub allows users to make Posts that can be displayed on the app. Currently,
...
+</p>
+If you choose the text type, enter the title and description as prompted; if you choose the link type, enter the title and the URL of the link and the address of the image as prompted.
+
+Click the Create to complete.
+<p align="center">
+ <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/posts/view-posts.png" />
+</p>
 
-To create a post, users must use the [createPost](../graphql/mutations.md#createPost) GraphQL mutation. There is currently no way to create posts using the UI, though this will come in the future.
+# Create Posts Using the GraphQL
```
Comment (yoonhyejin) [inline docs/posts.md:38]:
```suggestion
### Create Posts Using the GraphQL
```

**Comment by yoonhyejin** [comment]
@chardaway Sorry for the delay here! Made small suggestions on the headings

**Review by yoonhyejin — APPROVED** [review]

File: docs/posts.md [hunk 4]
```diff
@@ -13,9 +13,30 @@ DataHub allows users to make Posts that can be displayed on the app. Currently,
...
 Anyone can view Posts on the home page. To create Posts, a user must either have the **Create Global Announcements** Privilege, or possess the **Admin** DataHub Role.
 
-## Using Posts
+## Creating Posts
+### Create Posts Using the UI
+To create a post, first navigate to the Settings tab in the top-right menu of DataHub.
+Once you're on the Settings page, click 'Home Page Posts'.
+To create a new Posts, click '+ New Posts'.
+<p align="center">
+ <img width="70%"  src="https://raw.githubusercontent.com/chardaway/static-assets/73d813ee0ec6abdc0411661b27532a0300696ffa/imgs/posts/new-Post.png" />
```
Comment (yoonhyejin) [inline docs/posts.md:22]: I missed this part ; 
```suggestion
 <img width="70%"  src="raw.githubusercontent.com/datahub-project/static-assets/main/imgs/posts/new-post.png" />
```

File: docs/posts.md [hunk 5]
```diff
@@ -13,9 +13,30 @@ DataHub allows users to make Posts that can be displayed on the app. Currently,
...
+</p>
+DataHub currently supports two types of Post content. Posts can either be of type **Text** or **Link**. Click on 'Post Type' to switch between these types.
+<p align="center">
+ <img width="70%"  src="https://raw.githubusercontent.com/chardaway/static-assets/73d813ee0ec6abdc0411661b27532a0300696ffa/imgs/posts/PostType-Text.png" />
```
Comment (yoonhyejin) [inline docs/posts.md:26]:
```suggestion
 <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/posts/post-type-text.png" />
```

File: docs/posts.md [hunk 6]
```diff
@@ -13,9 +13,30 @@ DataHub allows users to make Posts that can be displayed on the app. Currently,
...
+</p>
+<p align="center">
+ <img width="70%"  src="https://raw.githubusercontent.com/chardaway/static-assets/73d813ee0ec6abdc0411661b27532a0300696ffa/imgs/posts/PostType-Link.png" />
```
Comment (yoonhyejin) [inline docs/posts.md:29]:
```suggestion
 <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/posts/post-type-link.png" />
```

**Review by yoonhyejin — CHANGES_REQUESTED** [review]
https://github.com/datahub-project/static-assets/pull/78

this PR should be merged first to include img assets under datahub repo

**Comment by chardaway** [comment]
Can I close the request "Update posts.md with some screenshot # 69" now? These files already exist, right?

**Comment by jjoyce0510** [comment]
Hi folks, checking in. Can we get this resolved? 

These are great docs!

**Review by yoonhyejin — APPROVED** [review]
LGTM!

**Pull request merged**